tests/sndrcv_relp_tls-cfgcmd.sh

```diff
@@ -55,7 +63,7 @@ if [ $ret == 0 ]; then
 	skip_test
 else
 	# Kindly check for a failed session
-	content_check "librelp error 10031" $RSYSLOG_DEBUGLOG
+	content_check "librelp: generic error: ecode 10031" $RSYSLOG_DEBUGLOG
```

Comment:
typo? ecode -> code ?

Reply:
its part of the debug output: 
https://github.com/rsyslog/librelp/blob/91aef5c53b92f93e47a36beee9c5937c0ec8365b/src/relp.c#L65